src/OpenTelemetry.Instrumentation.ElasticsearchClient/README.md

```diff
@@ -112,7 +112,8 @@ services.AddOpenTelemetry()
 that deprecates `NEST/Elasticsearch.Net`,
 brings native support for OpenTelemetry. To instrument it you need
 to configure the OpenTelemetry SDK to listen to the `ActivitySource`
-used by the library by calling `AddSource("Elastic.Clients.Elasticsearch.ElasticsearchClient")`
+used by the library by calling `AddSource("Elastic.Transport")` or
+`AddSource(Elastic.Transport.Diagnostics.OpenTelemetry.ElasticTransportActivitySourceName)`
```

Comment:
`Elastic.Clients.Elasticsearch.ElasticsearchClient` is still valid, for versions [8.0.0,8.10.0).
`Elastic.Transport` technically has broader impact than only Elastic.Client.Elasticsearch: https://www.nuget.org/packages/Elastic.Transport#usedby-body-tab

I think that we should be more precise on documentation.